src/main/java/ee/tuleva/onboarding/user/member/MemberRepository.java

```diff
@@ -0,0 +1,11 @@
+package ee.tuleva.onboarding.user.member;
+
+import org.springframework.data.jpa.repository.Query;
+import org.springframework.data.repository.CrudRepository;
+
+public interface MemberRepository extends CrudRepository<Member, Long> {
+
+  @Query("select max(m.memberNumber) from Member m")
+  Integer getMaxMemberNumber();
```

Comment:
perhaps rename getNextMemberNumber and do max + 1, as not to expose max member number outside this call at all?